### PR TITLE
Feature/python10

### PR DESCRIPTION
**Pull request by sdavrieux**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -9,13 +9,13 @@ authors = [
 ]
 
 [tool.poetry.dependencies]
-python = "~3.8"
+python = ">=3.8"
```
Comment (epureanudiana) [inline pyproject.toml]: why this change? 
if it's above 3.11, then it's not compatible with torch. 
maybe add also an upper bound?

Comment (sdavrieux) [reply pyproject.toml]: Because otherwise we cannot have 3.10 in OOR. Adding the upper bound.

**Review by epureanudiana — APPROVED** [review]
